### ISSUE TITLE
Small visual bug in Safari

### ISSUE THREAD

**Issue by dartess** [elrumordelaluz/reactour#490]
**Describe the bug**

The cross moved a little in Safari

**To Reproduce**
1. Open Safari
2. Open tour

**Expected behavior**

The cross is fully visible

**Screenshots**

<img width="812" alt="image" src="https://user-images.githubusercontent.com/15047511/185451702-bec852d0-445c-494a-ae5d-2643011ae75e.png">

**Additional context**

Can be fixed if add `display: block` to `svg`

**Comment by elrumordelaluz** [comment]
Thank you @dartess for the point!
Should be available now on `v3.0.1`

**Comment by dartess** [comment]
@elrumordelaluz Thanks for the quick reaction! Unfortunately, I'm not that fast.

Now version `v3.0.1` is not working:

```
WARNING in ../node_modules/@reactour/tour/dist/tour.esm.js 816:36-42
export 'Portal' (imported as 'Portal') was not found in '@reactour/utils' (possible exports: Observables, bestPositionOf, getInViewThreshold, getPadding, getRect, getWindow, inView, isHoriz, isOutsideX, isOutsideY, safe, smoothScroll, useElemRect, useIntersectionObserver, useRect)
```

`v3.1.0` works, but this original problem with cross still exists:

<img width="114" alt="image" src="https://user-images.githubusercontent.com/15047511/187211317-f8115bec-a1c5-467b-87a5-142cec4ed39e.png">

p.s. in demo https://reactour.vercel.app in safari now the cross went even lower; now it is not visible at all:

<img width="1134" alt="image" src="https://user-images.githubusercontent.com/15047511/187211270-9da9f599-da37-48ac-8484-041dc308dd82.png">

**Comment by elrumordelaluz** [comment]
Than you for pointing this out again, should be solved in `v3.1.1`
